backend/src/main/java/com/crm/exception/ResourceNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.crm.exception;
+
+public class ResourceNotFoundException extends RuntimeException {
```

Comment:
In Global Handler we catch "NoSuchElementException". I used this in other part of code where element is not found. Maybe we use this embedded exception instead creating new exception?

Reply:
Good point, corrected